docs/release-note-guide.md

```diff
@@ -0,0 +1,38 @@
+# Release Notes Language Style Guide
+
+When you write a release note for your pull request, make sure that your language style meets the following rules:
+
+1. Include `ACTION REQUIRED:` at the beginning if the change requires user action, e.g. deprecating or abandoning features:
+
+    - ACTION REQUIRED: Add the `timezone` support for [all charts]
+
+2. Every note starts with the "do" form of a verb. For example:
+
+    - Support backup to S3 with [Backup & Restore (BR)](https://github.com/pingcap/br)
+    - Fix Docker ulimit configuring for the latest EKS AMI
+
+3. Ensure no period at the end of note.
+
+4. Use a single backquote (``) to frame the following elements in your release notes:
+
+    - Custom Resource name
+    - Variable name
+    - Variable value
+    - Error message
+    - Single-line SQL statement
+    - Configuration item name
+    - Field name
+    - The element that needs emphasis (for example, table name, column name, schema name, etc.)
```

Comment:
Some of these examples are for tidb, do we need to change these to more suitable examples in our project?

Reply:
good catch

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/1530/commits/eab96a33014e6593d9ded8f5eb2f935125f9e277